service-impl/data-service-impl/hadoop-data-service/pom.xml

```diff
@@ -77,5 +77,155 @@
             <version>${jackson.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- This only needs to be here for AWS access-->
```

Comment:
These additions are not required as they are already included in the shaded-hadoop-client dependancy